src/pages/settings/AboutPage.js

```diff
@@ -11,17 +11,23 @@ import CONST from '../../CONST';
 import * as Expensicons from '../../components/Icon/Expensicons';
 import ScreenWrapper from '../../components/ScreenWrapper';
 import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
 import MenuItem from '../../components/MenuItem';
 import Logo from '../../../assets/images/new-expensify.svg';
 import {version} from '../../../package.json';
 import * as Report from '../../libs/actions/Report';
 import * as Link from '../../libs/actions/Link';
+import getPlatform from '../../libs/getPlatform/index.website';
+import compose from '../../libs/compose';
 
 const propTypes = {
     ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
 };
 
 const AboutPage = (props) => {
+    const shouldShowKeyboardShortcutsMenu = !props.isSmallScreenWidth && [CONST.PLATFORM.WEB, CONST.PLATFORM.DESKTOP].indexOf(getPlatform()) > -1;
```

Comment:
Sorry, due to our code guidelines we cannot use `getPlatform` here – using it is basically equivalent to using `Platform.select` or `Platform.OS`, which is prohibited in our app (consult the README, section `Cross Platform 99.9999%`. We should make this more explicit in our javascript style guide.

Anyways, the way you should implement this is:

1. Update the module structure as follows:

    ```
    src/pages/settings/AboutPage/
    ----| index.js   // Move all the code from the current AboutPage.js to here
    ----| platformSpecificMenuItems
    --------| index.js    // Export an array that contains the keyboard shortcuts menu item on wide screens or an empty array on narrow screens (mobile web)
    --------| index.native.js // Export an empty array
    ```
    
1. In the new `AboutPage/index.js`, import the `platformSpecificMenuItems` and add them to the array of menu items before rendering.

I know that this has the same result as what you've done here, but we've standardized on a code style that uses platform-specific file extensions instead of inline platform checks.

Reply:
Okay, I am aware of the `index.native.js`, etc. but wasn't aware that `getPlatform` cannot be used. I'll update the PR